snake_ladder.cpp

```diff
@@ -0,0 +1,81 @@
+#include<bits/stdc++.h>
+#include<time.h>
+using namespace std;
+int main()
+{
+    srand(time(0));
+    int snk, lad, per; // Number of snakes, ladders and persons
+    int ip, fp; // Initial and final position
+    cin >> snk; // taking input of number of snakes
+    unordered_map<int, int> snakes; //int to int map to find snake address
+    for(int i=0;i<snk;i++)
+    {
+        cin >> ip >> fp;
+        snakes[ip] = fp;
+    }
+    cin >> lad; // taking input of number of ladders
+    unordered_map<int, int> ladders; //int to int map to find ladder address
+    for(int i=0;i<lad;i++)
+    {
+        int ip, fp;
+        cin >> ip >> fp;
+        ladders[ip] = fp;
+    }
+    cin >> per;
+    vector<pair<string, int>> person; //string to int vector to keep count of present address of persons
+    for(int i=0;i<per;i++)
+    {
+        string name;
+        cin >> name;
+        person.push_back(make_pair(name,0));
+    }
+    int c = 0;
+    while(1)
+    {
+        int x = (rand()%6)+1; // Random dice value generator (1-6)
```

Comment:
should have been a method whose job would have to roll a dice and return a number